src/context.rs

```diff
@@ -811,6 +817,41 @@ impl Context {
         }
     }
 
+    /// Evicts persistent objects or allows certain transient objects
+    /// to be made peristent.
+    pub fn evict_control(
```

Comment:
Any chance you could pull the `evict_control` part into its own PR? There's a lot of changes around the handles and this doesn't directly fit in

Reply:
yeah absolutly